### ISSUE TITLE
Question: Confirmation of the pretrained checkpoint train split

### ISSUE THREAD

**Issue by Mattias421** [shivammehta25/Matcha-TTS#135]
Hello,

First of all, fantastic research, and thanks for providing a highly developed codebase!

I'm looking to use Matcha as a baseline and would like to know if the LJSpeech checkpoint provided is trained on the [taco split](https://github.com/NVIDIA/tacotron2/tree/master/filelists). I've had some problems in the past where I wasn't sure if the checkpoint was for research or for inference (and trained on the test set) ^w^

Also, if you have any other advice on using the checkpoint I'm happy to hear it :)

- Matt

**Comment by shivammehta25** [comment]
Hey Matt! Thank you for kinds words. Yea it was trained with the taco split. Looking forward to your research 😃 

I think it should be pretty straightforward just load it or use cli simply. Let me know if you face any issues.

Regards,
Shivam

**Comment by Mattias421** [comment]
Awesome, that's good to hear! Should hopefully have some interesting research soon :)
